pkgs/applications/version-management/git-and-tools/git-team/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, buildGoModule, fetchFromGitHub, installShellFiles }:
+buildGoModule rec {
+  pname = "git-team";
+  version = "1.7.0";
+
+  src = fetchFromGitHub {
+    owner = "hekmekk";
+    repo = "git-team";
+    rev = "v${version}";
+    sha256 = "0nl5j64b61jw4bkf29y51svjbndmqqrqx96yaip4vjzj2dx9ywm4";
+  };
+
+  vendorSha256 = "sha256-xJMWPDuqoNtCCUnKuUvwlYztyrej1uZttC0NsDvYnXI=";
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  postInstall = ''
+    go run main.go --generate-man-page > $out/${pname}.1
+    installManPage $out/${pname}.1
+
+    # Currently only bash completions are provided
+    go run main.go completion bash > $out/${pname}.bash
+    installShellCompletion --bash --name ${pname}.bash $out/${pname}.bash
+  '';
+
+  meta = with lib; {
+    description =
+      "Command line interface for managing and enhancing git commit messages with co-authors";
```

Comment:
```suggestion
    description = "Command line interface for managing and enhancing git commit messages with co-authors";
```

Reply:
Nixfmt breaks it into two lines because it succeeds 80 characters.

Reply:
updated

Reply:
Sandro can be very picky :P

Reply:
> Nixfmt breaks it into two lines because it succeeds 80 characters

Yeah thats stupid because we don't enforce the 80 line limit. Just not long enough is fine.

> Sandro can be very picky :P

description should always be a single line and not using multiline strings makes sure that it is obvious when it is to long and should be moved to longDescription.